### PR TITLE
Change text sale responsible to responsible

### PR DESCRIPTION
**Pull request by Ebed-meleck**
This change is for #7466.

Change Sales Manager to Manager

### CODE REVIEW

**Comment by jmcameron** [comment]
Please say what page this affects!

**Comment by Ebed-meleck** [comment]
> Please say what page this affects!

The receipt that is generated after patient billing

File: server/controllers/finance/reports/invoices/creditNote.handlebars [hunk 1]
```diff
@@ -145,7 +145,7 @@
 
   <div class="row">
     <div class="col-xs-5">
-      {{translate 'FORM.LABELS.SALE_RESPONSIBLE'}} <br />
```
Comment (jniles) [inline server/controllers/finance/reports/invoices/creditNote.handlebars:148]: If `SALE_RESPONSIBLE` is no longer used in the application, could you remove it from the `i18n` JSON files?  I did a quick search and couldn't find it used anywhere else in the application.

Comment (Ebed-meleck) [reply server/controllers/finance/reports/invoices/creditNote.handlebars:148]: All right

**Review by jmcameron — APPROVED** [review]
Looks good.
I tested it locally and all tests pass including end-to-end tests.

**Pull request merged**